pandas/tests/tools/test_to_datetime.py

```diff
@@ -2906,8 +2942,16 @@ def test_parsers(self, date_str, expected, cache):
 
         for res in [result1, result2]:
             assert res == expected
-        for res in [result3, result4, result6, result8, result9]:
+        for res in [result3, result4]:
             exp = DatetimeIndex([Timestamp(expected)])
+            exp = exp.as_unit(reso)
+            tm.assert_index_equal(res, exp)
+
+        for res in [result6, result8, result9]:
+            # These cases go through array_to_datetime, not array_to_strptime.
+            #  This means that as of GH#??? they do resolution inference
```

Comment:
Is there an issue now for this?

Reply:
looks like this comment is out of date and all the cases now do inference.  will update.